app/routes/page.php

```diff
@@ -153,11 +209,12 @@ function renderGuide(Slim $app, Guide $guide, Category $category)
     }
 
     if ($fetchContent) {
-        $markdown = file_get_contents('https://raw.githubusercontent.com/piwik/piwik/master/CHANGELOG.md');
+        $markdown = file_get_contents('https://raw.githubusercontent.com/piwik/piwik/3.x-dev/CHANGELOG.md');
```

Comment:
We can do this once we have develop branch for 2.X etc. It's a bit problematic currently to do this automatically since we will make `3.x` to `master` at some point and then we can change it but we basically never know to which version master points so it can be only kind of automated a bit later